src/providers/token-lists.provider.ts

```diff
@@ -134,8 +143,11 @@ export default {
       // methods
       toggleTokenList,
       isActiveList,
+      tokensListPromise,
     });
 
-    return () => slots.default();
+    return () => {
+      return slots.default();
+    };
```

Comment:
This change is required? Would be nice to keep the single line if possible.

Reply:
Done.